external_services/models/notification.py

```diff
@@ -22,11 +23,22 @@ def send(template_type: str, to_email: str, content: dict):
         """
         url = Config.NOTIFICATION_SERVICE_HOST + Config.SEND_ENDPOINT
         params = {"type": template_type, "to": to_email, "content": content}
+        current_app.logger.info(
+            f"Sending application to notification service. endpoint: '{url}',"
+            f" params: '{params}'."
+        )
         response = post_data(url, params)
+        current_app.logger.info(
+            "application sent to notification service with response:"
+            f" '{response}'."
+        )
         if response:
             return True
         raise NotificationError(
-            message="Sorry, the notification could not be sent"
+            message=(
+                "Sorry, the notification could not be sent for endpoint:"
+                f" '{url}', params: '{params}'."
+            )
```

Comment:
I think the sent log needs to move to after the if response check so we don't send a 'sent' log and then a 'could not send' log straight after in the scenario where we do not have a response :)

Reply:
good spot - refactored 👌